index.js

```diff
@@ -55,7 +55,13 @@ function main() {
           // Run goldbach here
           let max_int = Number(processNum.trim())
           gbf.set_max_number(max_int)
-          gbf.run_process();
+          let tupples = gbf.run_process();
+          console.log('We found %i Goldbach pair(s) for %i', tupples.length, max_int);
+          let index = 0;
+          while (index < tupples.length){
+            console.log('%i = %i + %i', max_int, tupples[0], tupples[1]);
```

Comment:
@MrBHerring This is should fix the issue of 3 and 7 showing up repeatedly
```suggestion
            console.log('%i = %i + %i', max_int, tupples[index][0], tupples[index][1]);
```